### PR TITLE
Added build instructions for macOS

### PR DESCRIPTION
**Pull request by y-mx-b**

### CODE REVIEW

**Comment by flxzt** [comment]
Thanks for taking the time to document the steps. :) Looks good to me, except for some minor corrections.

File: misc/building/rnote-macos-build.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+```
+
+Then, navigate into the `rnote` directory and following the build and
+installation steps.
+
+## Building and Installing the Project
+
+First, we have to setup the build directory. These steps will differ slightly
+from those listed in the
+[CONTRIBUTIONS.md](https://github.com/flxzt/rnote/blob/main/CONTRIBUTING.md#build-with-meson)
```
Comment (flxzt) [inline misc/building/rnote-macos-build.md:64]: I recently renamed this to BUILDING.md

**Comment by y-mx-b** [comment]
Alright, I've fixed those mistakes, is there anything else requiring fixing?

**Comment by flxzt** [comment]
I don't think so!

**Pull request merged**